mimic/resource.py

```diff
@@ -84,6 +91,14 @@ def advance_time(self, request):
             "now": seconds_to_timestamp(self.clock.seconds())
         })
 
+    @app.route("/mimic/v1.1/behaviors", branch=True)
```

Comment:
For Nova we had the endpoint  `url` as `http://localhost:8900/mimicking/NovaControlApi-52627e/ORD/v2/91111` and the `name` as `cloudServersBehavior`. For consistency can we have this endpoint be `/mimic/v1.1/IdentityControlApi` or such.

Also, we support v2.0 for Identity. Any particular reason this is v1.1.